CMakeLists.txt

```diff
@@ -278,7 +279,7 @@ else()
 endif()
 
 target_compile_definitions(ortcustomops PRIVATE  ${OCOS_COMPILE_DEFINITIONS})
-target_link_libraries(ortcustomops PRIVATE ${ocos_libraries})
+target_link_libraries(ortcustomops PUBLIC ocos_operators)
```

Comment:
Why is it public?

Reply:
Because C++ unit test depends on ocos_operator's headers. If here is PRIVATE, ortcustomops_test can't find some headers such as `ocos.h`